ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -88,39 +124,19 @@ open class BottomCommandingController: UIViewController {
     /// Indicates if the bottom commanding UI is hidden
     ///
     /// Changes to this property are animated.
-    @objc open var isHidden: Bool = false {
-        didSet {
-            if oldValue != isHidden && isViewLoaded {
-                if isInSheetMode {
-                    bottomSheetController?.isHidden = isHidden
-                } else if let bottomBarView = bottomBarView,
-                          let bottomConstraint = bottomBarViewBottomConstraint {
-                    if let animator = bottomBarHidingAnimator {
-                        animator.stopAnimation(true)
-                    }
-
-                    let springParams = UISpringTimingParameters(dampingRatio: Constants.BottomBar.hidingSpringDamping)
-                    let newAnimator = UIViewPropertyAnimator(duration: Constants.BottomBar.hidingSpringDuration, timingParameters: springParams)
-                    if isHidden {
-                        bottomConstraint.constant = -Constants.BottomBar.hiddenBottomOffset
-                        newAnimator.addCompletion { _ in
-                            bottomBarView.isHidden = true
-                        }
-                    } else {
-                        bottomBarView.isHidden = false
-                        bottomConstraint.constant = -Constants.BottomBar.bottomOffset
-                    }
-                    newAnimator.addAnimations { [weak self] in
-                        self?.view.layoutIfNeeded()
-                    }
-
-                    newAnimator.startAnimation()
-                    bottomBarHidingAnimator = newAnimator
-                }
-            }
+    @objc open var isHidden: Bool {
```

Comment:
isn't it bit confusing that there is IsHidden and then setishiddenwithanimation ?